WordPress/Classes/ViewRelated/Comments/CommentsViewController+Network.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+#import "CommentsViewController.h"
+@class WPTableViewHandler;
+
+
+/**
+ Fakes tableViewHandler as a protected property. Only classed importing this header will have access to it
```

Comment:
Typo here: "classed" should be "classes"

Reply:
😊